userspace/libsinsp/filterchecks.h

```diff
@@ -329,27 +329,48 @@ class sinsp_filter_check_thread : public sinsp_filter_check
 public:
 	enum check_type
 	{
-		TYPE_PID = 0,
-		TYPE_EXE,
+		TYPE_EXE = 0,
```

Comment:
I am not sure: given this is a public enum, isn't this an API break? 
I mean i know that nobody links libsinsp as a shared library, and we already broke API since 0.10.x release, but i am still unsure about this.

Reply:
Where is this used publicly? It looks to me that this is first mapped in order to the filterchecks defined as string and only used later, I believe I may be missing something

Reply:
It is not used at least by us, but it is exposed in a public header, right?
Perhaps i am wrong :)

Reply:
Also think it's a non-breaking change (definitely for Falco). We can add a release note with `!` for sure, just in case.